### DISCUSSION TITLE
Who won the 2023 UEFA ⚽🦶 League?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#952] [Galaxy Trivia] [answered]
**Bonus:** Which team had more passes per foul in the final?

**Comment by MrAlaminH** [comment]
Man City!!

**Comment by CelioAmaral** [comment]
O Manchester City Football Club ⚽! ; D

**Comment by peterbaccaro** [comment, marked as answer]
**Manchester City** won the 2023 UEFA Champions League.  It is the first time Manchester City have won the Champions League and they did it by beating Inter Milan 1-0 in the final.

Manchester City completed 512 passes and 11 fouls during the game.  So passes per foul was 512/11 = 46.5
Inter Milan completed 394 passes and 17 fouls during the game. So passes per foul was 394/17 = 23.17

Therefore **Manchester City had more passes per foul**.
